app/controllers/api/v1/cars_controller.rb

```diff
@@ -0,0 +1,87 @@
+class Api::V1::CarsController < ApplicationController
+  before_action :authenticate_user!, except: %i[index show]
+  before_action :find_car, only: %i[show update destroy]
+
+  # GET /cars
+  def index
+    if can? :read, Car
+      @cars = Car.all
+      if @cars.empty?
+        render_response(:not_found)
```

Comment:
- I'm not sure because I'm not able to run the project locally but, it looks like this render_response on application_controller.rb has the job to create the JSON response, now, according to this, the variable this send is `:not_found`. This could be a little confusing if we consider this to show a 4xx error because the request could be good (200) and empty at the same time, I invite you to:
- - Make sure it returns a 200 code when the request is good
- - Use ´empty record´ or similar in place to use `:not_found`, and use not found for 404 error 🐦